templates/dynamo/initial_last_updated_item.json.tpl

```diff
@@ -0,0 +1,8 @@
+{
+  "id": {
+    "S": "LastPolled"
+  },
+  "datetime": {
+    "S": "1970-01-01T00:00Z"
```

Comment:
Is this format ok? I thought it had to be same as the Preservica one (with the milliseconds and offset) e.g.
`2023-04-01T00:00:00.000000+0000`

Reply:
You're using `OffsetDateTime` in the code which is working with Preservica and this parses into an `OffsetDateTime` I've tried it and it's working ok. Once the event generator runs once, it'll be overwritten anyway.